README.md

```diff
@@ -1,22 +1,62 @@
-# Kinde ELixir SDK Starter Kit
+# Kinde Starter Kit - ELixir
+This is an Elixir template with [KindeAuth](https://kinde.com/docs/developer-tools/elixir-sdk/).
 
-Install erlang and elixir.
+## Register an account on Kinde
+To get started set up an account on [Kinde](https://app.kinde.com/register).
 
-Update :kinde_sdk config in `config/config.exs`.
+## Setup your local environment
+Clone this repo and within assets directory run this command `npm i`, then in the root directory execute `mix deps.get` to install the dependencies.
```

Comment:
Might make more sense to swap this around (mix deps.get vs npm i) since you land in the root directory first